emoji-modifier/index.ts

```diff
@@ -464,11 +464,15 @@ const parse = (message: string): ParseResult => {
   };
 };
 
-const runTransformation = async (message: string): Promise<Emoji | EmodiError | HelpRequest> => {
+const createResult = async (message: string): Promise<Emoji | EmodiError | HelpRequest> => {
```

Comment:
[COMMENT] If I were you, I'd choose a bit more descriptive name, like `buildResponse`. (Sorry for naming `result` `result`)